packages/backend/src/index.ts

```diff
@@ -55,13 +55,20 @@ import graphql from './plugins/graphql';
 import app from './plugins/app';
 import badges from './plugins/badges';
 import jenkins from './plugins/jenkins';
+import permission from './plugins/permission';
 import { PluginEnvironment } from './types';
+import { ServerPermissionClient } from '@backstage/plugin-permission-node';
 
 function makeCreateEnv(config: Config) {
   const root = getRootLogger();
   const reader = UrlReaders.default({ logger: root, config });
   const discovery = SingleHostDiscovery.fromConfig(config);
   const tokenManager = ServerTokenManager.noop();
+  const permissions = new ServerPermissionClient({
+    discoveryApi: discovery,
+    configApi: config,
+    serverTokenManager: tokenManager,
+  });
```

Comment:
should this also add the same modifications to the crete-app template and tell people in a changeset on that package how to make the corresponding change in their projects?

Reply:
We've decided to leave the changes out of create-app for now, but I've added changes to `docs/tutorials/backend-to-backend-auth.md` in this PR.

cc @Rugvip @mtlewis